docker/docs/index.html

```diff
@@ -85,7 +85,7 @@ <h2>Community</h2>
         >
       </li>
       <li>
-        <a href="http://pecanproject.github.io/Report_an_issue.html"
+        <a href="https://join.slack.com/t/pecanproject/shared_invite/enQtMzkyODUyMjQyNTgzLWEzOTM1ZjhmYWUxNzYwYzkxMWVlODAyZWQwYjliYzA0MDA0MjE4YmMyOTFhMjYyMjYzN2FjODE4N2Y4YWFhZmQ"
```

Comment:
Shouldn't this go to the new issues page?

```suggestion
        <a href="https://github.com/PecanProject/pecan/issues/new/choose"
```